src/main/java/io/cryostat/VerticleDeployer.java

```diff
@@ -48,20 +48,28 @@
 public class VerticleDeployer {
 
     private final Vertx vertx;
+    private final int poolSize;
     private final Logger logger;
 
     @SuppressFBWarnings(
             value = "EI_EXPOSE_REP2",
             justification = "vertx is externally mutable and that's fine")
-    public VerticleDeployer(Vertx vertx, Logger logger) {
+    public VerticleDeployer(Vertx vertx, int poolSize, Logger logger) {
         this.vertx = vertx;
+        this.poolSize = poolSize;
         this.logger = logger;
     }
 
     public Future deploy(Verticle verticle, boolean worker) {
         String name = verticle.getClass().getName();
         logger.info("Deploying {} Verticle", name);
-        return vertx.deployVerticle(verticle, new DeploymentOptions().setWorker(worker))
+        DeploymentOptions deploymentOptions = new DeploymentOptions();
+        deploymentOptions.setWorker(worker);
+        if (deploymentOptions.isWorker()) {
+            deploymentOptions.setWorkerPoolName(verticle.getClass().getSimpleName() + "-worker");
+            deploymentOptions.setWorkerPoolSize(poolSize);
+        }
+        return vertx.deployVerticle(verticle, deploymentOptions)
                 .onSuccess(id -> logger.info("Deployed {} Verticle [{}]", name, id))
```

Comment:
Would it be nice for the successfully deployed verticles to log their configuration? Like
```java
                .onSuccess(id -> logger.info("Deployed {} Verticle [{}] with deploymentOptions: {}", name, id, deploymentOptions.toJson().encodePrettily()))
```
If not, then I think 
```java
    @Provides
    @Singleton
    @Named(Variables.VERTX_POOL_SIZE)
    public static int provideVertxPoolSize(Environment env) {
        int poolSize = Math.max(1, Integer.parseInt(env.getEnv(Variables.VERTX_POOL_SIZE, "20")));
        logger.info("Vert.x workerPoolSize = {}, poolSize);
        return poolSize;
    }
```
would be nice just as small extra info as well.